pkg/whitesource/whitesource.go

```diff
@@ -317,50 +275,6 @@ func (s *System) GetProjectVulnerabilityReport(projectToken string, format strin
 	return respBody, nil
 }
 
-// GetOrganizationProductVitals
```

Comment:
why removing this? We should keep it.

Reply:
It was identical in implementation to GetProductsMetaInfo. I kept only one.

Reply:
ok